src/com/aca/homework/week2/maxmin/MainMaxMin.java

```diff
@@ -0,0 +1,23 @@
+package com.aca.homework.week2.maxmin;
+
+import java.util.Scanner;
+
+public class MainMaxMin {
+    public static void main(String[] args) {
+        System.out.println("Include 2 numbers");
+        long first = new Scanner(System.in).nextLong();
+        long second = new Scanner(System.in).nextLong();
+        System.out.println("Max is "+ compareNumbers(first,second).max + "\nMin is "+ compareNumbers(first,second).min);
```

Comment:
you can create a MaxMin object and reuse It to print the max and min instead of creation 2 objects